src/main/java/com/epam/pages/popup/SubjectPopup.java

```diff
@@ -124,4 +125,44 @@ public String getTheSearchLinePlaceholder() {
         return teacherSelectField.getDomProperty("placeholder");
     }
 
+    public boolean checkUIofAddTeachersPopup() {
+        logger.info("Check UI of add teacher popup - teachers select field, xButton and Save button");
+        return uiHelper.checkElementsAreDisplayed
```

Comment:
Please, follow the code style, write as this one - uiHelper.check..(
                                                                                       a
                                                                                       b
                                                                            );

Reply:
done